docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -93,12 +93,12 @@ Documents that contain information specific to the overall model and the pregnan
 2.2 Waves, GBD Rounds, and age groups
 -------------------------------------
 
-We will separate the implementation of the child model into two waves of updates. 
+We will separate the implementation of the child model into waves of updates. 
 In addition to other differences detailed in the next section:
 
 - Wave I will use GBD 2019 data, with the exception of using GBD 2021 data for child growth failure risk exposure and risk effects.
 
-- Wave II will use GBD 2021 data for the entire model.
+- Wave II and later will use GBD 2021 data for the entire model.
```

Comment:
We didn't actually get 2021 data into the wave II results that we presented in January, so maybe we should just update this to say that the 2021 update is coming in wave III?